build.gradle

```diff
@@ -26,7 +26,7 @@ task clean(type: Delete) {
 
 ext {
     // Sdk and tools
-    minSdkVersion = 11
+    minSdkVersion = 13
```

Comment:
This is not acceptable. We are targeting the minimum SDK 11. You have made changes as accordingly.
Please revert this change 13 to 11.